src/sidebar/store/modules/selection.js

```diff
@@ -314,6 +314,22 @@ function clearSelectedAnnotations() {
   return { type: actions.CLEAR_SELECTION };
 }
 
+/**
+ * Returns the annotation ID of the first annotation in `selectedAnnotationMap`.
+ *
+ * @return {string|null}
+ */
+function getFirstSelectedAnnotationId(state) {
+  const selection = state.selectedAnnotationMap || {};
+  for (const annId in selection) {
+    if (!selection.hasOwnProperty(annId)) {
```

Comment:
Wondering if this ever happens?

Reply:
Seems the short answer is no, it is always true or not present but maybe leave it for now until the data structure is potentially adjusted at sometime in the future.